docs/advanced/program-options.md

```diff
@@ -19,7 +19,7 @@ You can always get a list of all options by using the `-h` option.
 ## **Usage**
 
 ```
-dcrd [OPTIONS]
+dcrd <OPTIONS>
```

Comment:
Actually, I think the square brackets were correct there.  This is not a single value that the user replaces with their own stuff but a whole bunch of stuff.  Much like the way most man pages do it:
```
SYNOPSIS
       ls [OPTION]... [FILE]...
```